assets/stylesheets/kitten/components/tooltips/_tooltip.scss

```diff
@@ -8,103 +8,46 @@
-  $color: map-get($k-colors, 'background-1');
-
-  // Colors Lithium
-  $color-lithium: map-get($k-colors, 'primary-1');
-  $border-color-lithium: map-get($k-colors, 'primary-4');
-  $background-color-lithium: map-get($k-colors, 'background-1');
+  // Colors helium
+  $color-helium: map-get($k-colors, 'background-1');
+  $border-color-helium: map-get($k-colors, 'primary-1');
+  $background-color-helium: map-get($k-colors, 'primary-1');
 
   $font: 'source-sans';
   $font-size: -3;
   $line-height: 1.3;
 
   $arrow-size: k-px-to-rem(8px);
-  $tooltip-margin: k-px-to-rem(18px);
 
-  // To override __react_component_tooltip style.
-  .k-Tooltip .k-Tooltip__content {
+  .k-Tooltip {
     @include k-typographyFont($font);
     @include k-typographyFontSize($font-size, $line-height);
-  }
-
-  .k-Tooltip__content {
-    background-color: $background-color !important;
-    color: $color !important;
-    opacity: 1 !important;
-
-    padding: k-px-to-rem(20px) !important;
-
-    max-width: k-px-to-rem(340px) !important;
-    border-radius: k-px-to-rem(8px) !important;
 
-    &.place-top {
-      margin-bottom: $tooltip-margin !important;
+    padding-right: k-px-to-rem(20px);
+    padding-left: k-px-to-rem(20px);
+    width: k-px-to-rem(340px);
+    box-sizing: border-box;
 
-      &:after {
-        border-top-color: $background-color !important;
-        border-top-style: solid !important;
-        border-top-width: $arrow-size !important;
-        margin-left: -($arrow-size) !important;
-        bottom: k-px-to-rem(-8px) !important;
-        left: 50.3% !important;
-      }
-    }
-
-    &.place-right {
-      margin-left: $tooltip-margin !important;
-
-      &:after {
-        border-right-color: $background-color !important;
-        border-right-style: solid !important;
-        border-width: $arrow-size !important;
-        margin-top: -($arrow-size) !important;
-        left: k-px-to-rem(-8px) !important;
-        top: 52% !important;
-      }
-    }
-
-    &.place-bottom {
-      margin-top: $tooltip-margin !important;
-
-      &:after {
-        border-bottom-color: $background-color !important;
-        border-bottom-style: solid !important;
-        border-width: $arrow-size !important;
-        margin-left: -($arrow-size) !important;
-        top: k-px-to-rem(-8px) !important;
-        left: 50.3% !important;
-      }
-    }
-
-    &.place-left {
-     margin-right: $tooltip-margin !important;
+    background-color: $background-color-helium;
+    border: 1px solid $border-color-helium;
+    border-radius: k-px-to-rem(8px);
+    color: $color-helium;
+  }
 
-      &:after {
-        border-left-color: $background-color !important;
-        border-left-style: solid !important;
-        border-width: $arrow-size !important;
-        margin-top: -($arrow-size) !important;
-        right: k-px-to-rem(-8px) !important;
-        top: 52% !important;
-      }
-    }
+  .k-Tooltip__arrow {
+    position: absolute;
+    left: 8rem;
```

Comment:
Cette valeur me paraît très arbitraire, non ? J'imagine que celle qu'on va modifier en JavaScript, mais est-ce qu'on pourrait éviter de mettre une valeur ici pour l'instant ?

Par exemple on pourrait mettre `// TODO this should be set by JavaScript only` ici, ou alors appliquer ces `8rem` juste dans l'exemple ?